### PR TITLE
[release-v1.9] Fix inlined plugins from client-pkg

### PR DESCRIPTION
**Pull request by dsimansk**
Per our earlier chat. This's enough to have both lists initialized.

@rhuss

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -150,4 +152,12 @@ replace (
 	k8s.io/cli-runtime => k8s.io/cli-runtime v0.25.4
 	k8s.io/client-go => k8s.io/client-go v0.25.4
 	k8s.io/code-generator => k8s.io/code-generator v0.25.4
+
+	//Pin Knative versions
```
Comment (dsimansk) [inline go.mod:156]: Adding all packages for the convenience to avoid newer version from plugins afterwards.

File: cmd/kn/main.go [hunk 2]
```diff
@@ -68,6 +70,11 @@ func run(args []string) error {
 
 	pluginManager := plugin.NewManager(config.GlobalConfig.PluginsDir(), config.GlobalConfig.LookupPluginsInPath())
 
+	// Merge plugins that implements client-pkg Plugin type to the same manager instance
+	for _, p := range pkgplugin.InternalPlugins {
+		pluginManager.AppendPlugin(p)
+	}
+
```
Comment (dsimansk) [inline cmd/kn/main.go:77]: The important part!

**Comment by rhuss** [comment]
Thanks!

/approve
/lgtm

**Pull request merged**